packages/dd-trace/src/config.js

```diff
@@ -481,7 +481,8 @@ class Config {
     this._setValue(defaults, 'clientIpEnabled', false)
     this._setValue(defaults, 'clientIpHeader', null)
     this._setValue(defaults, 'crashtracking.enabled', true)
-    this._setValue(defaults, 'codeOriginForSpans.enabled', false)
+    this._setValue(defaults, 'codeOriginForSpans.enabled', true)
```

Comment:
Changing the default seems to be a breaking change. Should this land on the next major?

Reply:
It's not breaking because of the bug explained in the PR body.... there was a bug that meant that it was always `true` 😱

Reply:
Actually it wouldn't be breaking either way. It's no different from just landing the feature today and having it on by default from day one. The reason why it was `false` by default before was because we wanted to roll it out slowly.